src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -87,7 +84,6 @@ const myAppConfig = {
 
 Amplify.configure(myAppConfig);
 ```
-[question]: # (RENE, Minor detail: In line 84 above, should "ID" be all caps in both instances?)
 
```

Comment:
Q from Jim during his initial edit pass - In line 81 above, should "ID" be all caps in both instances?

Reply:
Yeah we should keep it consistent.